h2o-docs/src/product/migrating.rst

```diff
@@ -1,4 +1,30 @@
 .. _migration:
+.. include:: <isonum.txt>
+
+Migration Guide
+###############
+
+Migrating between H2O-3 versions
```

Comment:
@angela0xdata : I introduced this new section, and put it on top of the file as I believe that there are way more people today likely to migrate between h2o-3 versions than from h20-2 to 3.
Of course, any suggestion welcome.

Reply:
@seb-h2o I don't understand the include statement. What is <isonum.txt>?